django/contrib/admin/tests.py

```diff
@@ -231,3 +232,18 @@ def has_css_class(self, selector, klass):
             .find(klass)
             != -1
         )
+
+    def take_screenshot(self):
+        current_directory = os.getcwd()
+        screenshot_directory = os.path.join(current_directory, "screenshots")
+        if not os.path.exists(screenshot_directory):
+            os.makedirs(screenshot_directory)
+        screenshot_name = os.path.join(
+            screenshot_directory, f"{self._testMethodName}.png"
+        )
+        self.selenium.set_window_size(1366, 784)
```

Comment:
The default screen size of selenium was strange, so expanded it to a "standard" desktop size. This actually [might not be the most common](https://www.hobo-web.co.uk/best-screen-size/) one. Need to pick one and put an appropriate comment :+1: 
We can take multiple screenshots at different sizes. I would be tempted to either start with desktop or desktop and mobile rather than do too many too early. :thinking:

Reply:
Another idea, similar to how `runtests.py` has the cli argument `--selenium chrome,firefox` (list of browsers) maybe I should do something similar like `--screenshots 1366x784,1920x1200` (list of resolutions) and then you can run this locally and check different screen-sizes and the ci just does 1 or 2. :thinking: